blocks/rss-alexa-feature-block/features/rss/xml.js

```diff
@@ -0,0 +1,183 @@
+import PropTypes from 'fusion:prop-types'
+import Consumer from 'fusion:consumer'
+import get from 'lodash/get'
+import moment from 'moment'
+import getProperties from 'fusion:properties'
+import { resizerKey } from 'fusion:environment'
+import { BuildContent } from '@wpmedia/feeds-content-elements'
+import { generatePropsForFeed } from '@wpmedia/feeds-prop-types'
+import { buildResizerURL } from '@wpmedia/feeds-resizer'
+
+// const jsdom = require('jsdom')
+
+const jmespath = require('jmespath')
+
+const rssTemplate = (
+  elements,
+  {
+    channelTitle,
+    channelDescription,
+    channelPath,
+    channelCopyright,
+    channelTTL,
+    channelCategory,
+    channelLogo,
+    imageTitle,
+    imageCaption,
+    imageCredits,
+    itemTitle,
+    pubDate,
+    itemCategory,
+    includePromo,
+    includeContent,
+    resizerURL,
+    resizerWidth,
+    resizerHeight,
+    domain,
+    feedTitle,
+    feedLanguage,
+    rssBuildContent,
+  },
+) => ({
+  rss: {
+    '@version': '2.0',
+    channel: {
+      title: { $: channelTitle || feedTitle },
+      link: `${domain}`,
+      description: { $: channelDescription || `${feedTitle} News Feed` },
+      lastBuildDate: moment
+        .utc(new Date())
+        .format('ddd, DD MMM YYYY HH:mm:ss ZZ'),
+      ...(feedLanguage && { language: feedLanguage }),
+      ...(channelCategory && { category: channelCategory }),
+      ...(channelCopyright && {
+        copyright: channelCopyright,
+      }), // TODO Add default logic
+      ...(channelTTL && { ttl: channelTTL }),
+      ...(channelLogo && {
+        image: {
+          url: buildResizerURL(channelLogo, resizerKey, resizerURL),
+          title: channelTitle || feedTitle,
+          link: domain,
+        },
+      }),
+
+      item: elements.map((s) => {
+        let body, category
+        const url = `${domain}${s.website_url || s.canonical_url || ''}`
+        const img =
+          s.promo_items && (s.promo_items.basic || s.promo_items.lead_art)
+        return {
+          title: { $: jmespath.search(s, itemTitle) || '' },
+          link: url,
+          guid: s._id,
+          description: {
```

Comment:
You should delete this description tag and change the logic on 89 to be description.

Reply:
removed description